### ISSUE TITLE
Locker roadmap

### ISSUE THREAD

**Issue by marvinmarnold** [locker-labs/locker-ui#69]

**Comment by marvinmarnold** [comment]
@bilgute I shared a doc with you. let me know if it's missing any information. we should probably make a version for  docs.locker.money also.

**Comment by bilgute** [comment]
Yes, I got the doc yesterday. Thank you for reminding me. I will add my comment today.
